C_CPP/Fundamentals/Programming components, Structure & Keyword/Programming Components Structure_Keywords_C.md

```diff
@@ -0,0 +1,248 @@
+# Programming Components, Structure & Keywords
+
+## Components and Keywords
+<br>
+
+Let's look at the basic minimum of C programme components because if you understand the fundamental structure of a C programme, the language's other core building blocks will be straightforward to grasp.
+ 
+ ## Tokens in C
+
+ A token is a keyword, an identifier, a constant, a string literal, or a symbol, and a token is a keyword, an identifier, a constant, a string literal, or a symbol in a C programme. The following C statement, for example, is made up of five tokens:
+
+ ```c
+ printf("Hello, World! \n");
+ ```
+<br>
+
+ * **Semicolons**
+  <br>
+    The semicolon is a statement terminator in a C programme. That is, each statement must be terminated by a semicolon. It indicates the end of one logical entity.
+
+    ```c
+    printf("Hello, World!");
+    return 0;
+    ```
+    <br>
+
+* **Comments**
+  <br>
+    Comments are similar to help text in a C programme, but the compiler ignores them. They begin with the characters /* and end with the characters */, as illustrated below.
+
+    ```c
+    /* Program in C */
+    ```
```

Comment:
```suggestion
```c
    /* Program in C */
```
```

remove this space for syntax effect from all the code snippets